shared/util.go

```diff
@@ -90,6 +90,18 @@ func ToStringSlice(set mapset.Set) []string {
 	return result
 }
 
+// ToStringSet converts a typed slice to a set.
+func ToStringSet(slice []string) mapset.Set {
```

Comment:
This seems the same as `NewSetFromStringSlice`.

Reply:
Whoops. Looks like my naming choices aren't deterministic.